pkg/runtime/unstructured/converter.go

```diff
@@ -0,0 +1,472 @@
+/*
```

Comment:
I had hoped to get more than a 2x speedup but I'm ok with starting as is.  I would like a follow up opened to continue looking at this (I expected to see a 5x speedup).